src/Microsoft.AspNet.OData.Shared/Builder/EdmModelHelperMethods.cs

```diff
@@ -447,6 +447,7 @@ private static void ValidateActionOverload(IEnumerable<ActionConfiguration> conf
          
             // add dynamic dictionary property annotation for open types
             model.AddDynamicPropertyDictionaryAnnotations(edmTypeMap.OpenTypes);
+            model.AddInstanceAnnotationsDictionaryAnnotations(edmTypeMap.InstanceAnnotations);
 
```

Comment:
Should have separate comment, i.e.:
  // add instance annotation dictionary property annotation

Reply:
#Resolved